src/elements/hx-beacon/index.js

```diff
@@ -0,0 +1,14 @@
+import { HXElement } from '../../interfaces/HXElement/index.js';
+
+/**
+ * Defines behavior for the `<hx-beacon>` element.
+ *
+ * @extends HXElement
+ * @hideconstructor
+ */
+export class HXBeaconElement extends HXElement {
+    /** @override */
+    static get is () {
+        return 'hx-beacon';
+    }
```

Comment:
Make sure of verify that `<hx-beacon>` is listening for the `click` event.

Reply:
Changes `have been done, please review